api/handler/group/group_backup.go

```diff
@@ -239,7 +239,14 @@ func (h *BackupHandle) snapshot(ids []string, sourceDir string, force bool) erro
 		status := h.statusCli.GetStatus(id)
 		logrus.Debugf("service: %s is state: %v", service.ServiceAlias, service.IsState())
 		if !force && status != v1.CLOSED && service.IsState() { // state running service force backup
-			return fmt.Errorf("state app must be closed before backup")
+			versions, err := db.GetManager().VersionInfoDao().GetVersionByServiceID(id)
```

Comment:
if the component does not have versions, the status is undepoy? so judgment base status?